react/containers/PublishForm/view.jsx

```diff
@@ -70,16 +71,14 @@ class PublishForm extends React.Component {
     xhr.open('POST', uri, true);
```

Comment:
Cool.  Yeah, I replaced most xhr requests with fetch in the app.  I kept this one for the publish request because I wanted to set event listeners on the request and wasn't sure if I could do that with fetch.  I'll look into that.